Makefile

```diff
@@ -31,6 +32,14 @@ endif
 test: ## Executes tests
 		$(PHP) ./vendor/bin/phpunit
 
+.PHONY: test-extension
+test-extension: ## Executes tests for extension by first moving src dir
+		sudo sed -i -e 's/;extension=sanitizer.so/extension=sanitizer.so/g' $(INI_FILE)
+		mv src src_
+		$(PHP) ./vendor/bin/phpunit
+		mv src_ src
+		sudo sed -i -e 's/extension=sanitizer.so/;extension=sanitizer.so/g' $(INI_FILE)
```

Comment:
You should not use `sudo`. Just run `php -d extension=/path/to/the/ext.so script.php`